queries.sql

```diff
@@ -7,4 +7,40 @@ SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> ' Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- update tables with transactions
+BEGIN TRANSACTION;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
+
+-- DELETE COLUMN
+ALTER TABLE animals DELETE COLUMN species;
+
+-- update table in a transaction
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```

Comment:
- [ ] Kindly add :
```sql
 SELECT species from animals; 
```
before COMMIT to verify that change was made and after COMMIT to verify that change persists after commit

Reply:
well resolved! ✔️